apps/backend/src/lambdas/parser.ts

```diff
@@ -65,15 +83,56 @@ export const handler = async (event?: unknown, _context?: Context) => {
 
       // TODO [CCFPCM-318] Excluded LABOUR2 files that are DDF, needs implementation
       const finalParseList = parseList.filter(
-        (filename) => !filename?.includes('LABOUR2')
+        (filename) => !filename?.includes('LABOUR')
       );
+      appLogger.log('Creating daily upload for today if needed');
```

Comment:
Do we need to set a context for the logs? We should try to merge in your logger PR :D